src/backend/internal/controller/user/login.go

```diff
@@ -99,10 +111,7 @@ func (r *Resolver) Login(c *gin.Context) {
 	c.JSON(http.StatusOK, gin.H{
 		"status":              "success",
 		"session_token":       token,
-		"email":               existedUser.Email,
-		"id":                  existedUser.Id,
-		"name":                existedUser.Name,
-		"provider":            existedUser.Provider,
+		"data":                existedUser,
 		"profile_picture_url": GetProfilePictureUrl(existedUser.ProfilePictureKey),
```

Comment:
There's a mismatch between the Swagger API documentation for the /authen/v1/login endpoint and the actual response. The documentation states that user data, including the token, will be inside the data field. However, the implementation places profile_picture_url outside of data.

To align the documentation with the implementation, we should either update the Swagger @Success annotation to reflect the profile_picture_url being outside or adjust the response structure to include profile_picture_url within the data field. This will ensure consistency and clarity for API consumers.